definitions/tools/xenosplit.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Xenosplit"
+
+baseCommand: ["/bin/bash", "Xenosplit.sh"]
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 20000
+      tmpdirMin: 100000
+    - class: DockerRequirement
+      dockerPull: "mgibio/xenosplit:0.5"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'Xenosplit.sh'
+        entry: |
+            set -o pipefail
+            set -o errexit
+            
+            # Filtering the bam files and preparing them for xenosplit
+            /opt/samtools/bin/samtools sort -o graftsorted.bam $1
+            /opt/samtools/bin/samtools index graftsorted.bam
```

Comment:
Is this first sort and index necessary?  Can the `samtools view` filtering be done on an unsorted bam, since it's just looking at flags? (or does it complain if you try that)